app/controllers/application.js

```diff
@@ -4,7 +4,7 @@ const { Controller, computed, inject, get } = Ember;
 
 export default Controller.extend({
   session: inject.service(),
-
+  currentUser: computed.oneWay('session.session.content.authenticated'),
```

Comment:
I'm pretty sure that `session` should already have an `isAuthenticated` property on it...